src/game/behaviors/bowser.inc.c

```diff
@@ -1589,11 +1592,8 @@ void bowser_thrown_dropped_update(void) {
     // Set throw action and vel values
     cur_obj_get_thrown_or_placed(1.0f, 1.0f, BOWSER_ACT_THROWN);
     // Set swing speed based of angle
-    f32 swingSpd = o->oBowserHeldAngleVelYaw / 3000.0f * 70.0f;
     // If less than 0, reduce speed
-    if (swingSpd < 0.0f) {
-        swingSpd = -swingSpd;
-    }
+    f32 swingSpd = absf((o->oBowserHeldAngleVelYaw / 3000.0f) * 70.0f);
```

Comment:
i'd be interested to see if the parenthesis make it compile differently here, since x/3000.0f*70.0f could simplify down to x * (1/3000.0 * 70.0) right?